php-checks/src/main/resources/org/sonar/l10n/php/rules/php/S5779.html

```diff
@@ -0,0 +1,27 @@
+<p>PHPUnit assertions do throw a <em>PHPUnit\Framework\ExpectationFailedException</em> exception when they fail. This is how PHPUnit internally
+notices when assertions within testcases fail. However, if such an exception type or one of its parent types is captured within a try-catch block and
+not rethrown, PHPUnit does not notice the assertion failure.</p>
+<p>This check raises an issue on assertions within the <em>try</em> body of a <em>try-catch</em> block that do catch exceptions of the type
+<em>PHPUnit\Framework\ExpectationFailedException</em>, <em>PHPUnit\Framework\AssertionFailedError</em>, or <em>Exception</em>, and do not handle the
+variable holding the exception.</p>
+<h2>Noncompliant Code Example</h2>
+<pre>
+public function testA() {
+    try {
+        assertTrue(getValue()); // Noncompliant
+    } catch (\PHPUnit\Framework\ExpectationFailedException $e) {
+
+    }
+}
+</pre>
+<h2>Compliant Solution</h2>
+<pre>
+public function testB() {
+    try {
+        assertTrue(getValue()); // Compliant
+    } catch (\PHPUnit\Framework\ExpectationFailedException $e) {
+        assertEquals("Some message", $e-&gt;getMessage());
+    }
+}
+</pre>
+
```

Comment:
Maybe remove this empty line
```suggestion
```

Reply:
This seems to be added by the RSPEC tool. I see empty lines in the end of the other HTML files too.